includes/drawing.md

```diff
@@ -1,3 +1,4 @@
 > [!CAUTION]
-> - The `System.Drawing` namespace is not recommended for new development, due to not being supported within a Windows or ASP.NET service and it is not cross-platform. [ImageSharp](https://github.com/SixLabors/ImageSharp) and [SkiaSharp](https://github.com/mono/SkiaSharp) are recommended as alternatives.
-> - On Windows, `System.Drawing` depends on the GDI+ native library, which is shipped as part of the OS. Some Windows SKUs, like Windows Server Core or Windows Nano, don't include this native library as part of the OS. Exceptions will be thrown at run time because the library can't be loaded.
\ No newline at end of file
+> - The `System.Drawing` namespace is not recommended for new development, due to not being supported within a Windows or ASP.NET service. [ImageSharp](https://github.com/SixLabors/ImageSharp) and [SkiaSharp](https://github.com/mono/SkiaSharp) are recommended as alternatives.
```

Comment:
```suggestion
> - The `System.Drawing` namespace is not recommended for new development, because it's not supported within Windows services, ASP.NET Core services, and ASP.NET services. If you attempt to use `System.Drawing` classes within one of these application types, you may get run-time exceptions and diminished service performance. Recommended alternatives include [ImageSharp](https://github.com/SixLabors/ImageSharp), [SkiaSharp](https://github.com/mono/SkiaSharp), and [Windows Imaging Components](/windows/desktop/wic/-wic-about-windows-imaging-codec).
```